src/Kunstmaan/ApiBundle/Resources/config/services.yml

```diff
@@ -0,0 +1,32 @@
+services:
+    kunstmaan_api.controller.nodes:
+        class: Kunstmaan\ApiBundle\Controller\NodesController
+        arguments:
+            - "@doctrine.orm.default_entity_manager"
+        calls:
+            - [setViewHandler, ['@fos_rest.view_handler.default']]
+
+    kunstmaan_api.controller.pages:
+        class: Kunstmaan\ApiBundle\Controller\PagesController
+        arguments:
+            - "@doctrine.orm.default_entity_manager"
+            - "@kunstmaan_api.service.data_transformer"
+        calls:
+            - [setViewHandler, ['@fos_rest.view_handler.default']]
+
+    kunstmaan_api.service.data_transformer:
+        class: Kunstmaan\ApiBundle\Service\DataTransformerService
+
+    kunstmaan_api.service.transformers.page:
+        class: Kunstmaan\ApiBundle\Service\Transformers\PageTransformer
+        arguments:
+            - "@doctrine.orm.default_entity_manager"
+        tags:
+            - { name: kunstmaan_api.transformer, priority: -250 }
```

Comment:
The transformers are meant to change (add/edit) or transform the given object. The PageTransformer changes the NodeTranslation to an Api object so I gave it a lower priority to be able to do stuff before it is transformed. For instance when writing your own api you might want to transform it yourself to some other object and with the default priority you wouldn't be able to.